bower.json

```diff
@@ -19,13 +18,13 @@
     "paper-fab": "polymerelements/paper-fab#1 - 2",
     "paper-icon-button": "polymerelements/paper-icon-button#1 - 2",
     "iron-input": "polymerelements/iron-input#1 - 2",
-    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2"
+    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2",
+    "web-animations-js": "web-animations/web-animations-js#^2.2.0"
```

Comment:
`na-editor.html` uses `element.animate`. I never actually put the HTML import in there. I should do that...